src/web/index.js

```diff
@@ -741,13 +766,17 @@ class InsertNodes {
     this._prototypes = store.prototypes;
     this._types = store.types;
     this._sectionTemplatePath = store.sectionTemplatePath;
+
+    // Completely rerender prototype if any CSS modules classnames change
+    if (!isEqual(this._classNames, store.classNames)) {
+      const isInitialRender = !this._classNames;
+      this._classNames = store.classNames;
+
+      // Only rerender after initial render (when classnames is not falsy)
+      if (!isInitialRender) {
+        this.cycleModules();
+      }
+    }
   }
 }
-/* eslint-enable no-underscore-dangle */
-
-// Create a new instance of the InsertNodes class
-/*eslint-disable*/
-// Create object for modifiying the templates on the page and
-// initial first templates.
-const insert = new InsertNodes(modules, store);
-/*eslint-enable*/
+/* eslint-enable */
```

Comment:
Do you need to do an eslint-enable at the bottom? I thought it did that automatically :)